README.md

```diff
@@ -54,9 +60,109 @@ Kinvey CLI is distributed as an NPM package. After you install NPM, run the foll
 
    Deletes the specified profile or the active one if you don't specify a profile name.
 
+* `org list`
+
+    Lists all existing organizations within your Kinvey account.
+    
+* `org show [org]`
+
+    Shows detailed information about the specified organization or about the active one if you don't specify an organization. You can specify an organization by ID or name.
+
+* `org use <org>`
+
+    Sets the specified organization as active.
+    
+* `app create <name>`
+
+    Creates an application.
+    
+* `app list`
+
+    Lists all existing applications within your Kinvey account.
+    
+* `app show [app]`
+
+    Shows detailed information about the specified application or about the active one if you don't specify an application. You can specify an application by ID or name.
+
+* `app use <app>`
+
+    Sets the specified application as active.
```

Comment:
Not saying that it should be included here but I just want to mention that the app can again be specified by ID or name. Same applies for `org use <org>`.

Reply:
Included.